### ISSUE TITLE
Shortest_path algorihm?

### ISSUE THREAD

**Issue by Johhan85** [MicrosoftDocs/sql-docs#6222]
My fellow student and I are going to do a thesis where we will compare neo4j graph databases against sql server graph database. We intend to use Microsoft's shortest_path algorithm. Do you or anyone close to you know which algorithm it is? Is it Dijkstra?

/Best regards

Johan Hansson & Jimmy Nilsson
Dalarna University Sweden

[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 44d0352c-3838-e788-f99e-ba6171fe2acc
* Version Independent ID: 22c33c5e-8e6d-7bf8-cace-f59af7b5cc7f
* Content: [SHORTEST PATH (SQL Graph) - SQL Server](https://docs.microsoft.com/en-us/sql/relational-databases/graphs/sql-graph-shortest-path?view=sql-server-ver15)
* Content Source: [docs/relational-databases/graphs/sql-graph-shortest-path.md](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/relational-databases/graphs/sql-graph-shortest-path.md)
* Product: **sql**
* Technology: ****
* GitHub Login: @shkale-msft
* Microsoft Alias: **shkale**

**Comment by shkale-msft** [comment]
@Johhan85 Yes, it is Dijkstra's algorithm. Please feel free to reach out to me if you have questions on supported scenarios, how to best leverage graph, or how to tune the database, queries, etc. I will be happy to help.

Thanks,
shreya

**Comment by Johhan85** [comment]
@shkale-msft Thanks! This is appreciated, you will definitely hear from us. Do you know if it says somewhere that it is Dijkstra algorithm, because we will need a reference to this in our thesis. We looked at the documentation but coudnt find it.

**Comment by shkale-msft** [comment]
Hi @Johhan85, generally we don't publish details of our internal implementation, as it is subject to change at anytime. So, I cannot provide you an official document which will say that.
